### PR TITLE
Zipkin tracing

### PR DESCRIPTION
**Pull request by kkoz**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -72,8 +73,14 @@ configurations.all {
...
 dependencies {
+    compile 'io.prometheus:simpleclient:0.6.0'
+    compile 'io.prometheus:simpleclient_vertx:0.6.0'
+    compile 'io.zipkin.brave:brave:5.6.8'
+    compile 'io.zipkin.brave:brave-http:4.13.6'
+    compile 'io.zipkin.brave:brave-instrumentation-http:5.6.8'
+    compile 'io.zipkin.reporter2:zipkin-sender-okhttp3:2.10.0'
     compile 'ch.qos.logback:logback-classic:1.2.3'
-    compile ('com.glencoesoftware.omero:omero-ms-core:0.4.1') {
+    compile ('com.glencoesoftware.omero:omero-ms-core:0.4.1-SNAPSHOT') {
```
Comment (chris-allan) [inline build.gradle]: 0.4.2-SNAPSHOT

File: build.gradle [hunk 2]
```diff
@@ -72,8 +73,14 @@ configurations.all {
 }
 
 dependencies {
+    compile 'io.prometheus:simpleclient:0.6.0'
```
Comment (chris-allan) [inline build.gradle]: All these dependencies should be pulled in from the new version of ms-core 0.4.2-SNAPSHOT

File: build.gradle [hunk 3]
```diff
@@ -13,6 +13,7 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 repositories {
+    mavenCentral()
```
Comment (chris-allan) [inline build.gradle]: Do we need this?

Comment (kkoz) [reply build.gradle]: I thought the build had failed when I didn't include this but it seems to work fine now. I'll remove it.

**Comment by chris-allan** [comment]
Needs `0.4.x` merged in and conflicts resolved

**Pull request merged**